### ISSUE TITLE
Missing image in top editors results in alt text overlapping with username

### ISSUE THREAD

**Issue by minerCodeMaster** [outpoot/bliptext#44]
![Image](https://github.com/user-attachments/assets/ef3f2075-66a8-4abb-b251-ee3cd3cee915)

**Comment by asciidude** [comment]
![Image](https://github.com/user-attachments/assets/cdac691c-191e-4cc6-ba60-53d8d96100e6)
Profile pictures, at least mine, are also not getting properly linked. That, or it's outdated and won't update.

**Comment by face-hh** [comment]
Fixed in the latest commit
